### PR TITLE
Ferrari added

### PR DESCRIPTION
**Pull request by natalijamitic**
Referencing issue #3

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -1621,6 +1621,65 @@ const allSoloCars = {
...
+    '250 (non-LM)': {
+      'all': ['asp'],
+    },
+    '275': {
+      'all': ['asp'],
+    },
+    '308 Coupe & Spider': {
+      'all': ['asp'],
+    },
+    '308 & 328': {
```
Comment (Bjorn248) [inline common.js:1662]: You have `308 & 328` here and then `308 (all)` later. I think it makes sense to have separate 308 and 328 models, and then add the `cs` and `fp` classes to 308 while keeping only `cs` for 328.

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Looks good, just need to merge duplicate models into one.

File: common.js (hunk 1 above)
Comment (natalijamitic) [reply common.js:1662]: Yes you are right, will do asap

**Comment by Bjorn248** [comment]
Thanks!

**Pull request merged**